cnquery_test.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package cnquery
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetLatestVersion(t *testing.T) {
+	version, err := GetLatestVersion()
+
+	assert.NoError(t, err)
+	assert.NotNil(t, version)
+	assert.NotContains(t, cnqueryReleaseUrl, "github.com")
```

Comment:
check that it is `releases.mondoo.com`. We don't want this to be different than github but we explicitly want it to be `releases.mondoo.com`